### DISCUSSION TITLE
Atmos content is being missed

### DISCUSSION THREAD

**Discussion by klingonhighcmd** [meisnate12/Plex-Meta-Manager#2331] [Q&A]
I am using the latest version of Kometa in Docker. I have hundreds of Dolby Atmos Media files, but only 12 are correctly categorized and receive the appropriate poster modification regarding the Atmos audio format. As far as I can tell from the docs, Kometa only uses "audio_track_title" (that corresponds to Title in the metadata as listed by MediaInfo). Most of my media lists the Atmos descriptive title in the field called "Format_Commercial_IfAny" of the Audio metadata (this field is also referred to as "Commercial Title" by MediaInfo. If I copy the value of the "Format_Commercial_IfAny/Commercial Title" field to the "Title/audio_track_title" field, the media then appears in both my Atmos categories list and with the correct (Atmos) audio info on the poster. I am trying to avoid writing a script to do this. Is there a way to alter Kometa to check both fields to more accurately determine the type of audio contained in the media?

**Comment by chazlarson** [comment]
In the case of ATMOS, Kometa runs these regex:
```
          - key: truehd_atmos
            value: '(?i)^(?=.*\btrue[ ._-]?hd(\b|\d))(?=.*\batmos(\b|\d))'
...
          - key: plus_atmos
            value: '(?i)^(?=.*\b((dd[p+])|(dolby[ ._-]digital[ ._-]plus)|(e[ ._-]?ac3)\b))(?=.*\batmos(\b|\d))'
          - key: dolby_atmos
            value: '(?i)\batmos(\b|\d)'
```
<https://github.com/Kometa-Team/Kometa/blob/83dd84c2f2627a5f0f21af4009a1bc0e93ad420a/defaults/overlays/audio_codec.yml#L64>
Against:
```
      - audio_track_title.regex: <<regex_<<key>>>>
      - filepath.regex: <<regex_<<key>>>>
```
<https://github.com/Kometa-Team/Kometa/blob/83dd84c2f2627a5f0f21af4009a1bc0e93ad420a/defaults/overlays/audio_codec.yml#L98>

The `audio_track_title` is Kometa's name for what PlexAPI exposes as `extendedDisplayTitle` of an audio track.
<https://python-plexapi.readthedocs.io/en/latest/modules/media.html#plexapi.media.MediaPartStream>

The filepath is checked because most of these overlays expect TRaSH file naming; this sort of thing:
```
Zack Snyders Justice League (2021) [Remux-2160p HEVC DV HDR10 10-bit TrueHD Atmos 7.1]-FraMeSToR.mkv
```

Kometa doesn't access the actual media file itself at all.

If this "commercial title" is available via PlexAPI, of course support could be added, it's only typing, but I don't see it mentioned here:
<https://python-plexapi.readthedocs.io/en/latest/modules/media.html#plexapi.media.MediaPartStream>
